man/generate_man.rb

```diff
@@ -0,0 +1,47 @@
+require 'fileutils'
+def add_header_markup(time)
+  header = ""
+  time.times do
+    header += "-"
+  end
+  header += "\n"
+  header
+end
+
+def trim_colon(string)
+  string.include?(":") ? string.tr(':', '') : string
+end
+
+def available_plugins
+  available = `hammer --version`.split('*')
+  plugins = "Available Plugins\n"
+  plugins << "-----------------\n"
+  available.each do |plugin|
+    plugins << plugin.strip!
+    plugins << "\n\n"
+  end
+  plugins << "\n"
+  plugins
+end
```

Comment:
Also hint that there is `man <plugin-name>` with more detail about plugin content would help.